### PR TITLE
Fix Gravatar fallback behavior

### PR DESCRIPTION
**Pull request by jpuffer**
- do not pass fallback param to gravatar; instead, ask gravatar to always return 404 when there is no match. This allows unavatar to control the fallback behavior and to continue to look for other services for a match.
- fix behavior when no matching image is returned from any service. in that case, the `url` variable is undefined but no error is thrown so the catch is not triggered.  So we need to explicitly set url to the fallbackUrl in that case.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,6 +77,7 @@
     "hack": "latest",
     "husky": "latest",
     "lint-staged": "latest",
+    "ncu": "~0.2.1",
```
Comment (Kikobeats) [inline package.json]: why is this necessary?

Comment (jpuffer) [reply package.json]: As I was running the different npm scripts to try to get my local environment going, I noticed there was a call to this lib in the `npm run update` script, but it wasn't included as a dependency.  You know better what these scripts are for, so if this is useless I'll take it out and you can squash the commits when merging the PR

Comment (Kikobeats) [reply package.json]: oh, that's true, although the package name is `npm-check-updates`, let me update it 🙂

**Comment by Kikobeats** [comment]
@jpuffer I implemented your suggestions here
https://github.com/Kikobeats/unavatar/commit/fecfebcaa5e02f909de4da8a2cf904e105023adf#diff-eecebf1ac3704dc12f289082dfa812f6R18

I did it instead of merging the PR because I feel something in the core could be simplified and we can avoid pass the avatar fallback url per every service.

Now default avatar on gravatar service 🎉
